charts/argo-services/templates/workflows/notify-release/workflow.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: notify-release
+spec:
+  entrypoint: notify-release
+  arguments:
+    parameters:
+      - name: repositoryName
+      - name: commitSha
+      - name: environment
+  templates:
+    - name: notify-release
+      inputs:
+        parameters:
+        - name: repositoryName
+        - name: commitSha
+        - name: environment
+      script:
+        image: curlimages/curl
```

Comment:
No need to worry about it in this PR, but we should fetch these through ECR (pull-through cache maybe?) so that we don't get flakiness from DockerHub's rate-limiting.

Reply:
Good shout - on the to do list to look at!